asf_core_data/pipeline/data_joining/merge_proc_datasets.py

```diff
@@ -0,0 +1,206 @@
+# File: asf_core_data/pipeline/data_joining/merge_proc_datasets.py
+"""Join the EPC and MCS datasets."""
```

Comment:
I think we're missing something at the top here explaining what we are doing and what's different from the remaining scripts in the data_joining folder. I would also add a `README.md` to the folder to briefly explain these differences.

Reply:
Added a few lines but could probably be improved further. I'll keep the documentation of this script open.